medicines/doc-index-updater/src/delete_manager/mod.rs

```diff
@@ -241,10 +244,9 @@ mod test {
         ))
         .unwrap();
 
-        assert_eq!(
-            state_manager.get_most_recently_set_status(),
-            JobStatus::Accepted
-        );
+        let result =
+            block_on(state_manager.get_status(removeable_message.get_message().job_id)).unwrap();
```

Comment:
Is the `block_on` needed here? Not sure where the async aspect is coming from?

Reply:
`get_status` is async on the trait - test failed without this

Reply:
Ah I see, fair enough